test/oats/configs/grafana-dashboards.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: 1
```

Comment:
Do we use these dashboards in our tests?

Reply:
No.  I have removed this and cleaned out a few other things.